artifactory/services/go/publish.go

```diff
@@ -1,28 +1,122 @@
 package _go
 
 import (
+	"errors"
+	"fmt"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"net/http"
+	"net/url"
+	"strings"
+
 	"github.com/jfrog/jfrog-client-go/artifactory/services/utils"
 	"github.com/jfrog/jfrog-client-go/auth"
 	"github.com/jfrog/jfrog-client-go/http/jfroghttpclient"
+	clientutils "github.com/jfrog/jfrog-client-go/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/content"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/httputils"
+	"github.com/jfrog/jfrog-client-go/utils/version"
 )
 
-var publishers []PublishGoPackage
+const ArtifactoryMinSupportedVersionForInfoFile = "6.10.0"
```

Comment:
Since we have 6.10.0 as the minimal version for the info file addition, let's make 6.10.0 as the minimal versoon for the entire go support, and avoid this if statement from now on.